<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_edit_organ</class>
 <widget class="QDialog" name="Dialog_edit_organ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="list">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>10</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="show">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>show organs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_organ">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>edit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="old_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_head">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>head of organ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>new organ name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>old organ name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
